src/confdb/confdb.h

```diff
@@ -168,16 +168,6 @@
 
 /* SSH */
 #define CONFDB_SSH_CONF_ENTRY "config/ssh"
-#ifdef BUILD_SSH_KNOWN_HOSTS_PROXY
-/****************************************************************************
- * Don't forget to update src/config/cfg_rules.ini when these options are
- * definitively removed.
```

Comment:
You didn't read this comment and update `cfg_rules.ini` accordingly.

Reply:
Missed that file. Fixed now.